client/components/price/price-filter.js

```diff
@@ -0,0 +1,74 @@
+const frenchTouch = {
```

Comment:
Not really a fan of `touch` naming.

Reply:
I guess it was a joke in the original code